### ISSUE TITLE
MD038 edge case with internal backticks

### ISSUE THREAD

**Issue by nschonni** [DavidAnson/markdownlint#489]
The following doesn't trigger, although it does have internal padding space
```md
` HTMLMediaElement``.src `
```
This is something that was just a bug in terminating the code fence and re-opening it again, but it should still flag the MD038 since both the code spans have MD038 issues

https://dlaa.me/markdownlint/#%25m%60%20HTMLMediaElement%60%60.src%20%60

**Comment by nschonni** [comment]
Found some more of the odd ones searching over in mdn/content with this regex:
```
[^`]+``[^`]+"
```

**Comment by nschonni** [comment]
Actually, I'm thinking this might not be fixable, since I think the padding is actually according to CommonMark with this edge case, but unintentionally incorrect from their HTML -> MD migration

**Comment by nschonni** [comment]
Here are the ones I picked up in that repo https://github.com/mdn/content/pull/12325

**Comment by DavidAnson** [comment]
Like you said, CommonMark allows codes spans to have a single space at the beginning and end (and will remove both) in order to allow code that begins or ends with a backtick (though the code isn't required to do so). Your example above meets this criteria, but I can't tell from here if that's a bad conversion or deliberate.

https://spec.commonmark.org/0.30/#code-spans

**Comment by nschonni** [comment]
Yeah, looks like this is a case of https://spec.commonmark.org/0.30/#example-340
I think there were a few deliberate ones in the conversion and a few mistakes, but I don't think Markdownlint can actually tell the difference since they are valid in those examples
